src/config.cpp

```diff
@@ -30,8 +30,8 @@ namespace ovms {
 const uint AVAILABLE_CORES = std::thread::hardware_concurrency();
 const uint MAX_PORT_NUMBER = std::numeric_limits<ushort>::max();
 
-const std::string DEFAULT_REST_WORKERS_STRING{"24"};
-const uint64_t DEFAULT_REST_WORKERS = std::stoul(DEFAULT_REST_WORKERS_STRING);
+const uint64_t DEFAULT_REST_WORKERS = AVAILABLE_CORES * 4.0;
```

Comment:
I am not sure if this is not a bit too high for default setting. This would mean that we could in theory process 4*CPU core count requests.
Did you check perf results on CI machine for changed defaults?
Beside that looks good.

Reply:
this is based on Tensorflow Serving [defaults](https://github.com/tensorflow/serving/blob/d77c9768e33e1207ac8757cff56b9ed9a53f8765/tensorflow_serving/core/load_servables_fast.h#L48) and tested on machine with 80CPUs